src/test/java/com/gr4vy/sdk/Gr4vyClientTest.java

```diff
@@ -47,7 +64,7 @@ public void addBuyersTest() throws Gr4vyException, ApiException {
     	BuyersApi api = new BuyersApi(client.getClient());
     	BuyerRequest buyer = new BuyerRequest();
     	buyer.setDisplayName("Java Test");
-        Buyer response = api.addBuyer(buyer);
+        Buyer response = api.newBuyer(buyer);
```

Comment:
Does the README also need to be updated with `newBuyer`?

Reply:
Yes, README file updated accordingly. thanks!